setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 # no need to traverse these directories
-exclude = .git,__pycache__,data,data_resized,tools
+exclude = .git,__pycache__,data,data_resized,tools,ENV
```

Comment:
What is ENV?

Reply:
Sorry, At first time I created my virtualenv directory (/ENV) inside the project so had to add it for running flake. I missed it to remove.